src/phoenix/datasets/fixtures.py

```diff
@@ -347,7 +347,8 @@ def paths(self) -> Iterator[Tuple[DatasetRole, Path]]:
     ),
     prefix="unstructured/search/wiki",
     primary_file_name="queries.parquet",
-    reference_file_name="queries2.parquet",
+    # The reference file is intended purely for troubleshooting bad data. Un-comment for testing.
+    # reference_file_name="queries2.parquet",
```

Comment:
hiding this dataset for now @RogerHYang - just to make the use-case a bit more concrete